tests/integration/test_charm.py

```diff
@@ -24,26 +24,43 @@
 log = logging.getLogger(__name__)
 
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+KATIB_CONFIG = "volumes-web-app-viewer-spec-ck6bhh4bdm"
+CHARM_NAME = METADATA["name"]
+EXPECTED_CONFIG_MAP = {
```

Comment:
For improving readability, should we take this out of the test file and save it as a yaml file that we can read and process using `Path` and `yaml`?

Reply:
This is the returned data from `lightkube` is not in yaml format, it's the yaml conent converted to string

Reply:
ok, `lightkube` after all was returning the configMap like that because in the manifests it is rendered as a multi-line string and I actually missed that, so adding `| -` to the expected config map `yaml` file, resolved the assertion.

Reply:
+1